platform/windows/detect.py

```diff
@@ -178,13 +178,14 @@ def get_opts():
 
     return [
         ("mingw_prefix", "MinGW prefix", mingw),
-        # Targeted Windows version: 7 (and later), minimum supported version
+        # Targeted Windows version: 8 (and later), minimum supported version
         # XP support dropped after EOL due to missing API for IPv6 and other issues
         # Vista support dropped after EOL due to GH-10243
+        # Windows 7 support dropped after EOL due to GH-95630
```

Comment:
DX12 is not supported on Windows 7 in any case, so there's no need to change min. Windows version, it's not core change and we are loading DX dlls dynamically. Windows 7 can still use OpenGL.

If it's done because of `WS_EX_NOREDIRECTIONBITMAP`, it can be redefined like other version specific constants in the `display_server_windows.h`:
```c++
#ifndef WS_EX_NOREDIRECTIONBITMAP
#define WS_EX_NOREDIRECTIONBITMAP 0x00200000L
#endif
```

Reply:
One other option I see would be to force win8 when compiling with d3d12 support, but not sure how to do that with scons.

Reply:
> The direct composition API is not available on Windows 7 or earlier.

Neither is DX12, so it will never be used. But we can still compile a single executable that will support both DX12 and GL on Windows 7.

Reply:
Well it's either upgrading to win8/10 or not supporting transparency with d3d12. I have not found a way to enable transparency with d3d12 besides using direct composition and the validation layers hint at there not being one.

Reply:
We can load `DCompositionCreateDevice` of adding `dcomp` to LIBS as we do with other DX dlls (for the same reason).

Reply:
Ah I see what you mean now. Ye that would also be possible I guess. I will adjust the PR when I find some time.

Reply:
Updated to use LoadLibrary instead